packages/divider/src/lib/Divider.tsx

```diff
@@ -0,0 +1,7 @@
+import { Divider as MuiDivider, DividerProps as MuiDividerProps } from '@mui/material';
+
+export type DividerProps = Omit<MuiDividerProps, 'children' | 'textAlign'>;
+
+export const Divider = ({ ...rest }: DividerProps): JSX.Element => {
```

Comment:
I think since we're spreading everything might as well do `(props: DividerProps)` and then we can spread that onto `MuiDivider`